Projects/Scripts/Misc/Email.cs

```diff
@@ -105,18 +105,18 @@ private static async void SendAsync(MimeMessage message)
     {
 
       DateTime now = DateTime.UtcNow;
-      string messageID = $"<{now.ToString("yyyyMMdd")}.{now.ToString("HHmmssff")}@{EmailServer}>";
+      string messageID = $"<{now.ToString("yyyyMMdd")}.{now.ToString("HHmmssff")}@{EMAIL_SERVER}>";
       message.Headers.Add("Message-ID", messageID);
-      message.From.Add(FromAddress);
+      message.From.Add(FROM_ADDRESS);
 
-      for (var count = 1; count <= m_retryCount; count++)
+      for (var count = 1; count <= RETRY_SEND_EMAIL_COUNT; count++)

```

Comment:
Let's make this more like regular for loops:
`for (int i = 0; i < RETRY_SEND_EMAIL_COUNT; i++)`